test.js

```diff
@@ -102,4 +102,23 @@ describe('dat-gateway', function () {
       throw e
     })
   })
+
+  it('should open a websocket to /peers', function () {
+    const req = http.get('http://localhost:5917/bunsen.hashbase.io/')
```

Comment:
Should you be waiting for the request to complete before trying to open the socket?

Reply:
Yeah, sounds good, I'll correct this when I get a chance.